stdlib/public/Concurrency/TaskCancellation.swift

```diff
@@ -67,10 +67,26 @@ import Swift
 /// Therefore, if a cancellation handler must acquire a lock, other code should
 /// not cancel tasks or resume continuations while holding that lock.
 @available(SwiftStdlib 5.1, *)
-#if !$Embedded
-@backDeployed(before: SwiftStdlib 6.0)
-#endif
+@_alwaysEmitIntoClient
```

Comment:
going with just AEIC those; we keep changing concurrency annotations and those methods are so small that AEIC for them is going to be fine anyway